docs/sdkx-react-native/release-notes-react-native.mdx

```diff
@@ -21,6 +21,14 @@ import {
 
 # Release Notes
 
+## 10.2.4
+
+_June 2nd, 2023_
+
+** Release Notes: **
+
+- Added an API to pause/resume the In-app notifications. For more details, refer:[Notifications](/sdkx-react-native/notifications-ios/#pause-in-app) page.
```

Comment:
For more details, refer [Notifications](url) page.

Reply:
Updated.